.github/workflows/pre-commit.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 env:
   TERRAFORM_DOCS_VERSION: v0.16.0
+  TFLINT_VERSION: v0.44.1
```

Comment:
Using a version avoids a few GitHub API calls which is currently causing issues with some workflows. Looks like they might have changed some of the rates, not sure why this is now an issue but just using a version seems to resole this